tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js

```diff
@@ -377,10 +377,22 @@ describe('Chaos experiments manager tests', function () {
             }
         };
         const chaosJobExperimentId = uuid();
+        const jobId = uuid();
         it('should call k8s connector and write to db', async function() {
-            await manager.runChaosExperiment(kubernetesJobConfig, chaosJobExperimentId);
+            const mappedChaosExperiment = {
+                ...chaosJobExperimentId,
+                metadata: {
+                    ...chaosJobExperimentId.metadata,
+                    labels: {
+                        app: 'predator',
+                        job_id: jobId
+                    }
+                }
+            };
+
+            await manager.runChaosExperiment(mappedChaosExperiment, jobId, chaosJobExperimentId);
```

Comment:
You are not supposed to call the manager with the mapped chaos experiment - should be the input k8s config and the manager should map it.

Reply:
done